src/overlays/actors/ovl_En_Test/z_en_test.c

```diff
@@ -28,22 +28,247 @@ const ActorInit En_Test_InitVars = {
     (ActorFunc)EnTest_Draw,
 };
 
-#endif
+void func_80862B70(EnTestStruct* arg0_) {
```

Comment:
Why the underscore?
```suggestion
void func_80862B70(EnTestStruct* arg0) {
```

Reply:
Because I use it in the function.

Reply:
I would prefer to avoid pointless underscores on variables like that. Could you use another name instead?

Reply:
I have no idea what to call it, what do I call it?

Reply:
I'd say just rename the argument to arg0, while the temp variable name as `ptr`. Especially since the temp isn't actually `arg0` but just elements of it.